bin/tictactoe

```diff
@@ -1,3 +1,10 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "\t X O X O X O X O X O X O X O X O X O X O X"
+puts "\t\t Welcome to Tic Tac Toe!"
+puts "\t X O X O X O X O X O X O X O X O X O X O X"
+
+game = Game.new
+game.start
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.

Furthermore, by splitting the interface (`TicTacToeCLI`) from the implementation (`Game`) of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.